resources/check-layout-th.js

```diff
@@ -208,12 +208,21 @@ var testNumber = 0;
 var highlightError = false; // displays outline around failed test element.
 var printDomOnError = true; // prints dom when test fails.
 
-window.checkLayout = function(selectorList, callDone = true)
+window.checkLayout = async function(selectorList, callDone = true)
 {
     if (!selectorList) {
         console.error("You must provide a CSS selector of nodes to check.");
         return;
     }
+
+    // Allow for a delay before checking the layout
+    // Polyfills can set this variable to give themselves time to apply changes
```

Comment:
@mfreed7 [said](https://github.com/oddbird/wpt/commit/6f92b20ff9a16c159f93af655f468d29071becd6#r97831002):

> It'll be important to expand this comment just a bit. It should be made clear that regular WPTs should not use this variable, and it should be reserved for polyfill testing. Perhaps a check should be added to the test runner that asserts this variable is undefined unless the --inject-script flag is present? I'm not sure if there's an observable way to check for --inject-script from here.

Reply:
@mfreed7 There are hooks that allow us to [modify the response](https://github.com/oddbird/wpt/blob/bb01a96bd8aae099e8e35fed1a7c4a37d3352c27/tools/serve/serve.py#L263-L270) in any way want when injecting scripts. I imagine we could add another global variable or DOM marker to indicate the presence of injected scripts.